app/EvenCountColorCode.js

```diff
@@ -0,0 +1,44 @@
+import { ColorPair } from "./colorPair.js";
+export class EvenCountColorCode {
+    MajorColorNames = [
+        "WHITE", "RED", "BLACK", "YELLOW", "VIOLET"
+    ];
+    MinorColorNames = [
+        "BLUE", "ORANGE", "GREEN", "BROWN", "SLATE"
+    ];
+    COLOR_PAIRS = {}
+    COLOR_PAIRS_KEY_VALUE_REVERSED = {}
+
+    constructor(){
+        let minorColorSize = this.MinorColorNames.length
+        let majorColorSize = this.MajorColorNames.length
+        for (let i = 0; i < majorColorSize; i++){
+            for (let j = 0; j < minorColorSize; j++){
+                this.COLOR_PAIRS[(i*majorColorSize)+(j + 1)] = [this.MajorColorNames[i],this.MinorColorNames[j]]
+                this.COLOR_PAIRS_KEY_VALUE_REVERSED[[this.MajorColorNames[i],this.MinorColorNames[j]]] = (i*majorColorSize)+(j + 1)
```

Comment:
Variable naming: The word `REVERSED` is subjective - what is reverse for you can be straight for me.
Hint: This is a map from something to something else. Try naming the variable in terms of the mapping

Reply:
Changed the name to PAIRNUMBER_COLORPAIR_MAP and COLORPAIR_PAIRNUMBER_MAP